### PR TITLE
Fix GitMergeBot build

### PR DESCRIPTION
**Pull request by cston**

### CODE REVIEW

**Comment by cston** [comment]
@dotnet/roslyn-infrastructure, @tannergooding please review.

[Thanks @jasonmalinowski for the help.]

**Comment by cston** [comment]
@dotnet-bot retest this please

File: src/Tools/Github/GitMergeBot/project.json [hunk 1]
```diff
@@ -10,6 +10,6 @@
     "net46": { }
   },
   "runtimes": {
-    "win7": { }
+    "win": { }
```
Comment (tannergooding) [inline src/Tools/Github/GitMergeBot/project.json]: This is going to need to be 'win7' when we do the further work of making this project use the Roslyn props/targets files

**Comment by cston** [comment]
@dotnet-bot retest this please

**Pull request merged**